packages/core/admin/server/src/services/homepage.ts

```diff
@@ -99,8 +99,10 @@ const createHomepageService = ({ strapi }: { strapi: Core.Strapi }) => {
         locale: document.locale,
         updatedAt: new Date(document.updatedAt),
         title: document[meta.mainField ?? 'documentId'],
-        publishedAt: meta.hasDraftAndPublish ? new Date(document.publishedAt) : null,
+        publishedAt:
+          meta.hasDraftAndPublish && document.publishedAt ? new Date(document.publishedAt) : null,
```

Comment:
@markkaylor this was the status bug. If D&P was enabled, we would set publishedAt even if the document was not published. At as I found out:
![screenshot 2024-12-11 at 09 49 44](https://github.com/user-attachments/assets/952be27f-772c-4dda-b10e-034ef278aeec)

Reply:
Nice good catch, just tested and look like it's working